tests/WE2E/test_configs/verification/config.MET_verification_only_vx.yaml

```
@@ -22,5 +22,6 @@ workflow:
     - parm/wflow/verify_det.yaml
 
 verification:
+  VX_MASK: ["CONUS","EAST","EAST_ROCKIES"]
```

Should be `EAST_OF_ROCKIES`, rather than `EAST_ROCKIES`.

```suggestion
  VX_MASK: ["CONUS","EAST","EAST_OF_ROCKIES"]
```